be/src/agent/task_worker_pool.cpp

```diff
@@ -707,6 +708,14 @@ void TaskWorkerPool::_publish_version_worker_thread_callback() {
                 std::this_thread::sleep_for(std::chrono::seconds(1));
             }
         }
+        if (res == OLAP_ERR_ROWSET_VERSION_NOT_CONTINUOUS) {
+            std::unique_lock<std::mutex> worker_thread_lock(_worker_thread_lock);
+            // push back to task queue if publish version not continuous
+            // make sure publish version task run in order
+            _tasks.push_back(agent_task_req);
+            _worker_thread_condition_variable.notify_one();
```

Comment:
This thread is working, so we donot need notify here.

Reply:
fixed